bundle-config-loader.js

```diff
@@ -2,38 +2,61 @@ module.exports = function (source) {
     this.cacheable();
     const { angular = false, loadCss = true, registerModules = /(root|page)\.(xml|css|js|ts|scss)$/ } = this.query;
 
+    const hmr = `
+        if (module.hot) {
+            const fileSystemModule = require("tns-core-modules/file-system");
+            const applicationFiles = fileSystemModule.knownFolders.currentApp();
+
+            global.__hmrLivesyncBackup = global.__onLiveSync;
+            global.__onLiveSync = function () {
+                console.log("HMR: Sync...");
+                require("nativescript-dev-webpack/hot")(__webpack_require__.h(), (fileName) => applicationFiles.getFile(fileName));
+            };
+
+            global.__hmrRefresh = function({ type, module }) {
+                global.__hmrNeedReload = true;
+                setTimeout(() => {
+                    if(global.__hmrNeedReload) {
+                        global.__hmrNeedReload = false;
+                        global.__hmrLivesyncBackup({ type, module });
+                    }
+                });
+            }
+
+            global.__hmrInitialSync = true; // needed to determine if we are performing initial sync
```

Comment:
I think this one is not needed. 
It was used before in a old version of [hot-loader-helper](https://github.com/NativeScript/nativescript-dev-webpack/blob/172e49866299a47fd726e14af227465ca43ca48f/hot-loader-helper.js#L7).Not used anymore though.